lib/uplink-gomobile/build.sh

```diff
@@ -11,23 +11,36 @@ then
       exit 1
 fi
 
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
 # setup tmpdir for testfiles and cleanup
 TMP=$(mktemp -d -t tmp.XXXXXXXXXX)
+GOPATH_SYSTEM=$GOPATH
```

Comment:
Use `$(go env GOPATH)`. `GOPATH` isn't alyways set anymore.

Reply:
thanks! I wasn't sure how to find it without system env set.